picoscope/darwin_utils.py

```diff
@@ -27,7 +29,13 @@ def LoadLibraryDarwin(library):
             FileNotFoundError
         except NameError:
             FileNotFoundError = IOError
-        if not Path(PICO_LIB_PATH).is_dir():
+
+        PICO_LIB_PATH = None
+        for path in PICO_LIB_PATHS:
+            if Path(path).is_dir():
+                PICO_LIB_PATH = path
+
+        if PICO_LIB_PATH is None:
```

Comment:
The for/else clause is perfect here. 

http://book.pythontips.com/en/latest/for_-_else.html